### DISCUSSION TITLE
Instructions to create packages using frb v2

### DISCUSSION THREAD

**Discussion by Demoevery** [fzyzcjy/flutter_rust_bridge#1897] [Q&A] [answered]
I wish to create package that includes the rust code. So user can simply "pub get" the package and compile rust code in their own machine or if possible use precompiled files. 
I created an example inside a new package and added depedency using "path: ../". Cargokit tries to find files as if it was a flutter project then fails.  I think this is not yet implemented for frb v2.
Similar discussion I found.
https://github.com/fzyzcjy/flutter_rust_bridge/discussions/819#discussioncomment-4478665
Am I missing something?

**Comment by fzyzcjy** [comment, marked as answer]
For v2, one way is to firstly create a cargokit package, and then configure flutter_rust_bridge.yaml to point to corresponding paths, and run things like `flutter_rust_bridge_codegen generate` as normal.
